vector/src/main/java/im/vector/app/features/home/room/list/actions/RoomListQuickActionsEpoxyController.kt

```diff
@@ -48,54 +46,38 @@ class RoomListQuickActionsEpoxyController @Inject constructor(
+        bottomSheetRoomPreviewItem {
+            id("room_preview")
+            avatarRenderer(host.avatarRenderer)
+            matrixItem(roomSummary.toMatrixItem())
+            stringProvider(host.stringProvider)
+            colorProvider(host.colorProvider)
+            izLowPriority(roomSummary.isLowPriority)
+            izFavorite(roomSummary.isFavorite)
+            settingsClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.Settings(roomSummary.roomId)) }
+            favoriteClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.Favorite(roomSummary.roomId)) }
+            lowPriorityClickListener { host.listener?.didSelectMenuAction(RoomListQuickActionsSharedAction.LowPriority(roomSummary.roomId)) }
+        }
 
-            // Notifications
-            bottomSheetDividerItem {
-                id("notifications_separator")
-            }
+        // Notifications
+        bottomSheetDividerItem {
+            id("notifications_separator")
         }
 
-        if (isV2) {
-            notificationViewState.notificationOptions.forEach { notificationState ->
-                val title = titleForNotificationState(notificationState)
-                radioButtonItem {
-                    id(notificationState.name)
-                    titleRes(title)
-                    selected(notificationViewState.notificationStateMapped() == notificationState)
-                    listener {
-                        host.listener?.didSelectRoomNotificationState(notificationState)
-                    }
+        notificationViewState.notificationOptions.forEach { notificationState ->
+            val title = titleForNotificationState(notificationState)
+            radioButtonItem {
+                id(notificationState.name)
+                titleRes(title)
+                selected(notificationViewState.notificationStateMapped() == notificationState)
+                listener {
+                    host.listener?.didSelectRoomNotificationState(notificationState)
                 }
             }
-        } else {
-            val selectedRoomState = notificationViewState.notificationState()
-            RoomListQuickActionsSharedAction.NotificationsAllNoisy(roomSummary.roomId).toBottomSheetItem(0, selectedRoomState)
-            RoomListQuickActionsSharedAction.NotificationsAll(roomSummary.roomId).toBottomSheetItem(1, selectedRoomState)
-            RoomListQuickActionsSharedAction.NotificationsMentionsOnly(roomSummary.roomId).toBottomSheetItem(2, selectedRoomState)
-            RoomListQuickActionsSharedAction.NotificationsMute(roomSummary.roomId).toBottomSheetItem(3, selectedRoomState)
         }
 
-        if (showFull) {
-            RoomListQuickActionsSharedAction.Leave(roomSummary.roomId, showIcon = !isV2).toBottomSheetItem(5)
-        }
+        RoomListQuickActionsSharedAction.Leave(roomSummary.roomId, showIcon = !true).toBottomSheetItem(5)
```

Comment:
So please also revert this change

Reply:
(ignore)

Reply:
I think more cleanup can be done in the fun `RoomListQuickActionsSharedAction.toBottomSheetItem(...)`, there is some dead code now.

Reply:
good catch! removed https://github.com/vector-im/element-android/pull/4627/commits/ab04e4078460c462d7ccd8f41c2cb479e524644b